core/domain/fs_domain.py

```diff
@@ -321,7 +322,8 @@ def get(self, filepath):
             raise IOError('File %s not found.' % (filepath))
         return file_stream.read()
 
-    def commit(self, filepath, raw_bytes, mimetype=None):
+    def commit(
+        self, filepath: str, raw_bytes: bytes, mimetype: str =' ') -> None:
```

Comment:
```suggestion
    def commit(
        self, filepath: str, raw_bytes: bytes, mimetype: str =' '
    ) -> None:
```
Also, why is `None` changed to `' '`?

Reply:
Because str type did not accept None.

Reply:
Then change the type to `Optional[str]`

Reply:
or maybe use `str | None`